src/sentry/replays/endpoints/project_replay_summarize_breadcrumbs.py

```diff
@@ -128,7 +129,31 @@ def fetch_error_details(project_id: int, error_ids: list[str]) -> list[ErrorEven
         return []
 
 
-def generate_error_log_message(error: ErrorEvent) -> str:
+def fetch_feedback_details(feedback_id: str, project_id: int | None = None):
+    """
+    Fetch user feedback associated with a specific feedback event ID.
+    """
+    if project_id is None:
+        return None
+
+    try:
+        node_id = Event.generate_node_id(project_id, event_id=feedback_id)
+        event = nodestore.backend.get(node_id)
+
+        return GroupEvent(
+            category="feedback",
+            id=feedback_id,
+            title="User Feedback",
+            timestamp=event.get("timestamp", 0.0) * 1000,  # feedback timestamp is in seconds
```

Comment:
```suggestion
            timestamp=event.get("timestamp", 0.0) / 1000,  # feedback timestamp is in seconds
```
event ts is ms right

Reply:
the feedback event is in seconds so to turn it into ms we need to multiply by 1000. same with errors which we do in the endpoint as well

Reply:
Oh my bad got it backward!